### PR TITLE
Strapi update to v4.0.6

### PR DESCRIPTION
**Pull request by zeddotes**
- updated Strapi to v4.0.6

### CODE REVIEW

File: config/server.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 module.exports = ({ env }) => ({
   host: env('HOST', '0.0.0.0'),
   port: env.int('PORT', 1337),
+  app: {
+    keys:['myKeyA', 'myKeyB']
+  }
```
Comment (var-kyle) [inline config/server.js:6]: I can't find any reference to these, what are they?

Comment (zeddotes) [reply config/server.js:6]: Good eye. If you remove those, Strapi complains about a missing config, which is obviously not documented anywhere. It suggests adding a `keys` array in a new (or unfamiliar) object called `app`.

**Review by var-kyle — APPROVED** [review]
The upgrade works, but sadly still doesn't fix styling issues when deploying in docker containers. And of course we get new configurations that aren't mentioned anywhere, but it still works.

**Review by var-kyle — APPROVED** [review]
Re-approving from that merge

**Pull request merged**